rotkehlchen/tests/conftest.py

```diff
@@ -110,3 +112,12 @@ def profiler(request):
 
     if profiler_instance is not None:
         profiler_instance.stop()
+
+
+def requires_env(allowed_envs: List[str]):
+    """Conditionally run tests if the environment is in the list of alloweds"""
```

Comment:
Please carefully check your PRs for typos and/or broken English

```suggestion
    """Conditionally run tests if the environment is in the list of allowed environments"""
```